universe/Effect.cpp

```diff
@@ -681,13 +681,8 @@ std::string SetShipPartMeter::Description() const {
 std::string SetShipPartMeter::Dump() const {
     std::string retval = DumpIndent();
     switch (m_meter) {
-        case METER_DAMAGE:              retval += "SetDamage";              break;
-        case METER_SPEED:               retval += "SetSpeed";               break;
```

Comment:
This is more grooming / stale code removal. There were previous plans to have a bunch of part meters to track things like fighter speed, structure, or detection. Presently the only part meter that matters for anything is capacity, so is the only part meter setting effect that needs to be supported.